TomTomSDKExamples/Examples/UseCase/BasicNavigationMenu.swift

```diff
@@ -0,0 +1,172 @@
+//  © 2023 TomTom NV. All rights reserved.
+//
+//  This software is the proprietary copyright of TomTom NV and its subsidiaries and may be
+//  used for internal evaluation purposes or commercial use strictly subject to separate
+//  license agreement between you and TomTom NV. If you are the licensee, you are only permitted
+//  to use this software in accordance with the terms of your license agreement. If you are
+//  not the licensee, you are not authorized to use this software in any manner and should
+//  immediately return or destroy it.
+
+import SwiftUI
+
+struct BasicNavigationMenu: View {
```

Comment:
Why not simply `NavigationMenu`?

Reply:
The "BasicNavigationController" naming was aligned with the Android example "BasicNavigationActivity" in the first version. Later this "BasicNavigationMenu" name was chosen for the new screen. As mentioned, "NavigationMenu" is simpler, so it's updated!